### PR TITLE
add plugin_version parameter for helm_plugin module

### LINKED ISSUE #157: Add option to set version for helm_plugin
##### SUMMARY

Add option to set version of plugin to install

When you install helm plugins we can set a version of the plgin to install like so:
`helm plugin install --version 0.8.0 https://github.com/helm/helm-2to3`
We should add this option so we can keep those plugins updated as required.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME

helm_plugin

##### ADDITIONAL INFORMATION

```yaml

```

### PR DESCRIPTION
**Pull request by abikouo**
##### SUMMARY

closes #157 

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by abikouo** [comment]
recheck

File: plugins/modules/helm_plugin.py [hunk 1]
```diff
@@ -24,14 +24,15 @@
   state:
     description:
       - If C(state=present) the Helm plugin will be installed.
+      - If C(state=latest) the Helm plugin will be updated. Added in version 2.2.0.
```
Comment (Akasurde) [inline plugins/modules/helm_plugin.py:27]:
```suggestion
      - If C(state=latest) the Helm plugin will be updated. Added in version 2.3.0.
```

File: plugins/modules/helm_plugin.py [hunk 2]
```diff
@@ -40,6 +41,13 @@
         machine and not on Ansible controller.
       - Required only if C(state=present).
     type: str
+  plugin_version:
+    description:
+      - Plugin version to install. If this is not specified, the latest version is installed.
+      - Ignored when C(state=absent) or C(state=latest).
+    required: false
+    type: str
+    version_added: "2.2.0"
```
Comment (Akasurde) [inline plugins/modules/helm_plugin.py:50]:
```suggestion
    version_added: "2.3.0"
```

File: plugins/modules/helm_plugin.py [hunk 3]
```diff
@@ -142,6 +163,9 @@ def main():
 
     if state == 'present':
         helm_cmd_common += " install %s" % module.params.get('plugin_path')
+        plugin_version = module.params.get('plugin_version')
+        if plugin_version is not None:
+            helm_cmd_common += " " + "--version=" + plugin_version
```
Comment (Akasurde) [inline plugins/modules/helm_plugin.py:168]:
```suggestion
            helm_cmd_common += " --version=%s" % plugin_version
```

**Review by Akasurde — CHANGES_REQUESTED** [review]
Rest LGTM

**Comment by abikouo** [comment]
@Akasurde changes requested have been resolved, could you please review/approve ? Thanks

**Review by gravesm — APPROVED** [review]

**Pull request merged**